player/tests.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from .models import Player, BallGame
+import pytest
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass'
+    )
+
+
+@pytest.fixture
+def player(user):
+    return Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer
+    )
+
+
+def test_create_player(player, user):
+    assert player.user == user
+    assert player.birth_date == '1990-01-01'
+    assert player.favorite_ball_game == BallGame.Soccer
+
+
+def test_update_player(player):
+    player.favorite_ball_game = BallGame.Basketball
+    player.save()
+    updated_player = Player.objects.get(pk=player.pk)
+    assert updated_player.favorite_ball_game == BallGame.Basketball
+
+
+def test_delete_player(player):
+    player.delete()
```

Comment:
Does the user should be deleted as well? You have `on_delete=django.db.models.deletion.CASCADE` on the user field on the player definition. Does this mean that we should expect the user to be deleted? If so, need to test that.

Reply:
It's actually the other way around. The player should be deleted if the underlying user is being deleted. We should test that as well.

Reply:
> Does the user should be deleted as well? You have `on_delete=django.db.models.deletion.CASCADE` on the user field on the player definition. Does this mean that we should expect the user to be deleted? If so, need to test that.

I am kinda getting lost, i am pretty sure when user is deleted the player should be deleted as well, beacsue of the oneToOne field connection they have.. but this should be also true when player deleted the same user sould be deleted as well.

Reply:
As far as I understand it, the cascade stuff means that deleting the user should also delete the player. Add a test to verify that.